packages/react/src/combobox/ComboboxRenderItem.tsx

```diff
@@ -1,10 +1,8 @@
-import { Fragment, isValidElement, ReactNode, useMemo } from 'react';
-import { boxPalette, tokens } from '../core';
-import { fontGrid } from '../core/utils/fontGrid';
+import { Fragment, isValidElement, ReactNode } from 'react';
 
 export type ComboboxRenderItemProps = {
-	/** The label of the item. */
-	itemLabel: string;
+	/** The label of the item. ReactNode is a private, character-separated label and for internal use only. */
```

Comment:
This is weird, but I don't know how else to handle this. When consumers do a custom item, they have to pass the string label, which I then override with the split version when rendering in the base component.